environment.md

```diff
@@ -59,23 +75,16 @@ The priority of the transports are Direct file IO, SSH2, FTP PHP Extension, FTP
 
 ## Database
 
-WordPress stores content, configuration, and other information inside of a database. The database for a WordPress website is where all of the WordPress website's user-defined data is stored. WordPress is primarily designed to use a MySQL or MySQL-related database server; WordPress officially only supports MySQL or MariaDB, a drop-in replacement for MySQL.
-
-### MySQL
-
-MySQL is a widely used relational database server. MySQL comes in both open-source and commercial distributions. Either distribution should work with MySQL. The commercial distribution has additional features not found in the open-source distribution; however, WordPress does not require or use the additional features. It is designed to run on either the open-source or the commercial distribution.
-
-### MariaDB
-
-MariaDB is a drop-in replacement for MySQL supported by WordPress. It's a fork of the open-source distribution of MySQL. MariaDB was originally created to maintain a more open-source version of MySQL, but it has grown into its own relational database server alternative to MySQL with features and changes not found in MySQL. Despite its differences, MariaDB is still a fully compatible replacement for MySQL and can generally seamlessly replace MySQL.
+For data storage, WordPress uses systems based on MySQL distributions, such as MySQL, MariaDB or Percona. All three systems are compatible with WordPress, so at this moment any of the three can be used for its operation.
 
-### Percona
+The use of these versions is recommended, both for performance and security reasons, although previous versions usually work without problems:
 
-Percona server is a drop-in replacement for MySQL, focused on performance. Although it's a drop-in replacement for MySQL, WordPress does not officially support Percona.
+* [MariaDB](https://mariadb.org/) 10.4.x / 10.3.x / 10.2.x
+* [MySQL](https://dev.mysql.com/downloads/mysql/) 8.0.x / 5.7.x
+* [Percona MySQL Server](https://www.percona.com/software/mysql-database/percona-server) 8.0.x / 5.7.x
 
-### Recommended Versions
+## How do I know which version I have?
 
-WordPress recommends the following versions for your Database:
+By choosing a web server, PHP and a database you will get your WordPress base installation working at first.
 
-*   MySQL 5.6 or greater
-*   MariaDB 10.1 or greater
\ No newline at end of file
+If you already have WordPress installed and want to know what you have, we recommend using the WordPress Community Plugin called [Health Check & Troubleshooting](https://wordpress.org/plugins/health-check/) (more [help for this plugin](https://make.wordpress.org/support/handbook/appendix/troubleshooting-using-the-health-check/)). If you have WordPress 5.2+, the system already has check tools to have that information in the Site Health section (at Tools in the menu).
```

Comment:
Can you list these two options in the opposite order? Maybe say that the tools are available, then note that if it's an older version of WordPress, the plugin can be used?

Reply:
Something like...

_If you have WordPress 5.2+, the Admin already has check tools to have that information in the Site Health section (at Tools in the menu).

If you have an older version, you can activate the Site Health section installing the WordPress Community Plugin called [Health Check & Troubleshooting](https://wordpress.org/plugins/health-check/) (more [help for this plugin](https://make.wordpress.org/support/handbook/appendix/troubleshooting-using-the-health-check/))._

Reply:
👍